pom.xml

```diff
@@ -74,11 +74,6 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
-            <dependency>
```

Comment:
Mark,

Is there documentation somewhere that guided you for this removal, is there a maven target that helps you handle these "duplicates", or do you "just" know this is part of the BOM?
Thanks.